doc/source/serve/deployment.rst

```diff
@@ -309,266 +309,3 @@ Feel free to open new github issues and contribute more storage backends!
 
 .. _serve-monitoring:
```

Comment:
need to move this reference as well. 

and also can we add a "Moved! notes here to redirect user to new section"

Reply:
ah yeah will move the reference

i don't think we need that, they can just search for it

Reply:
ah right, forgot that our search is good now